app/presenters/place_presenter.rb

```diff
@@ -0,0 +1,29 @@
+class PlacePresenter < ContentItemPresenter
+  PASS_THROUGH_DETAILS_KEYS = %i(
+    introduction
+    more_information
+    need_to_know
+    place_type
+  ).freeze
+
+  PASS_THROUGH_DETAILS_KEYS.each do |key|
+    define_method key do
+      details[key.to_s] if details
+    end
+  end
+
+  def initialize(content_item, places = nil)
+    @content_item = content_item
+    @places = places if places
+  end
+
+  def places
+    if @places
+      @places.map do |place|
+        place['text']    = place['url'].truncate(36) if place['url']
```

Comment:
I believe its a style thing, e.g. take a look at the result in the following screenshot.

![truncated-url](https://cloud.githubusercontent.com/assets/1130010/23860177/105c69f4-07fe-11e7-9068-248a735ff070.png)